ws_sdk/app.py

```diff
@@ -691,6 +691,17 @@ def get_tokens_from_name(self,
 
         return ret
 
+    def get_name_from_token(self, token: str = None) -> str:
```

Comment:
@meramsey  , what is the use case for this function ? 
Why not calling directly to `get_scope_name_by_token(token)` ?

Reply:
cause to someone new trying to use the library its much easier to see this helper function then understanding they need to think in the perspective of scopes to know to request that. 

I agree it seems unnecessary at first glance, but there are other helper functions in the lib that are already as superfluous if you view this as superfluous. I found it useful so figured others might too vs hunting down what stuff is called. Figured if you guys merged upstream i can remove my local function ideally.

Reply:
Hi @meramsey, apricate your suggestion.
We had an internal discussion about this and for now, we decided that we will add documentation about the get_scope various methods so it will have a much clearer usage.